build/vega-lite-schema.json

```diff
@@ -7892,28 +7892,36 @@
         },
         "labelFont": {
           "description": "The font of the header label.",
-          "type": "string"
+          "type": [
```

Comment:
Hmm, the schema generator should be updated to not generate arrays with a single type. Can you file an issue?

Reply:
https://github.com/vega/ts-json-schema-generator/issues/337